torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -69,7 +69,11 @@ def get_model(self, *, dl_kwargs=None) -> Wav2Vec2Model:
         Args:
             dl_kwargs (dictionary of keyword arguments): Passed to :func:`torch.hub.load_state_dict_from_url`.
         """
-        model = wav2vec2_model(**self._params)
+        model_type = self._params.pop("model_type", None)
```

Comment:
@sgrigory @nateanl Since model_type is not model configuration, can we move it outside of `_params`, and define it directly on Bundle class?